package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "nodejsmp",
+  "version": "1.0.0",
+  "description": "Homework for nodejs mentoring program",
+  "main": "app.js",
+  "scripts": {
+    "start": "npm run build && nodemon ./dist/app.js --exec babel-node",
```

Comment:
Consider checking the work of `nodemon` and maybe `watch-ts` script - after changes made there's no reloading of code which makes watcher and nodemon useless